### ISSUE TITLE
[BUG] TestingV2Source path does not match package declaration

### ISSUE THREAD

**Issue by jlowe** [NVIDIA/spark-rapids#3585]
The TestingV2Source.scala file is in the `com/nvidia/spark/rapids/tests/datasourcev2` directory yet the package declaration in the file doesn't line up:
```scala
package com.nvidia.spark.rapids.tests.datasourcev2.parquet
```
Note the extra "parquet" that's missing from the directory where the source is located.

**Comment by akbhuker** [comment]
hey, can I have this issue?
I'm quite new to this so I might need some guidance

**Comment by tgravescs** [comment]
yes that is fine, its really just moving the file to be under the directory that matches the package.  So  right now its in ./integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/datasourcev2/TestingV2Source.scala and it needs to move to com/nvidia/spark/rapids/tests/datasourcev2/parquet/TestingV2Source.scala

Feel free to put up a pull request for it if you could like.  I would suggest running the datasourcev2_read_test.py integration tests to make sure its still picked up properly.

**Comment by jlowe** [comment]
Thanks for offering to help, @akbhuker!  Please check out the [contributing guide](https://github.com/NVIDIA/spark-rapids/blob/branch-21.10/CONTRIBUTING.md) if you have not already, and note in particular the [section on signing off on your work](https://github.com/NVIDIA/spark-rapids/blob/branch-21.10/CONTRIBUTING.md#sign-your-work).
